docs/data/xrootd/install-cms-xcache.md

```diff
@@ -0,0 +1,242 @@
+Administrators may provide additional configuration by placing files in `/etc/xrootd/config.d/1*.cfg` (for files that need to be processed BEFORE the OSG configuration) or `/etc/xrootd/config.d/9*.cfg` (for files that need to be processed AFTER the OSG configuration).
+
+You _must_ configure every variable in `/etc/xrootd/10-common-site-local.cfg`.
+
+The mandatory variables to configure are:
+
+- `set rootdir = /mnt/stash`: the mounted filesystem path to export.  This document refers to this as `/mnt/stash`.
+- `set resourcename = YOUR_RESOURCE_NAME`: the resource name registered with the OSG for example ("T2_US_UCSD")
+
+!!! note
+    XRootD can manage a set of independent disk for the cache. So you can modify file `90-cms-xcache-disks.cfg` and add the disks there then `rootdir` just becomes a place to hold symlinks.
+
+
+### Ensure the xrootd service has a certificate
+
+The service will need a certificate for reporting and to authenticate to CMS AAA.
+The easiest solution for this is to use your host certificate and key as follows:
+
+1. Copy the host certificate to `/etc/grid-security/xrd/xrd{cert,key}.pem`
+1. Set the owner of the directory and contents `/etc/grid-security/xrd/` to `xrootd:xrootd`:
+
+        :::console
+        root@host # chown -R xrootd:xrootd /etc/grid-security/xrd/
+
+!!! note
+    You must repeat the above steps whenever you renew your host certificate.
+    If you automate certificate renewal, you should automate copying as well.
+    For example, if you are using Certbot for Let's Encrypt, you should write a "deploy hook" as documented
+    [here](/security/host-certs/lets-encrypt/#pre-and-post-renewal-hooks).
+
+!!! note
+    You must also register this certificate with the [CMS VOMS](https://voms24.cern.ch:8443/voms/cms/)
+
+Configuring Optional Features
+-----------------------------
+
+### Adjust disk utilization
+
+To adjust the disk utilization of your cache, create or edit a file named `/etc/xrootd/config.d/90-local.cfg`
+and set the values of `pfc.diskusage`.
+
+```
+pfc.diskusage 0.90 0.95
+```
+
+The two values correspond to the low and high usage water marks, respectively.
+When usage goes above the high water mark,
+the XRootD service will delete cached files until usage goes below the low water mark.
+
+### Modify the storage access settings at a site
+
+In order for CMSSW jobs to use the cache at your site you need to modify the `storage.xml` and create the following rules
+
+```
+# Portions of /store in xcache
+  <lfn-to-pfn protocol="direct" destination-match=".*"
+    path-match="/+store/(data/.*/.*/NANOAOD/.*)"
+    result="root://yourlocalcache:1094//store/$1"/>
+  <lfn-to-pfn protocol="direct" destination-match=".*"
+    path-match="/+store/(mc/.*/.*/NANOAODSIM/.*)"
+    result="root://yourlocalcache:1094//store/$1"/>
+```
+
+
+!!! note
+    If your are installing a multinode cache then instead of `yourlocalcache:1094` url should be changed for `yourcacheredirector:2040`
```

Comment:
```suggestion
    If your are installing a [multinode cache](#installing-a-multinode-cache) then instead of `yourlocalcache:1094` url should be changed for `yourcacheredirector:2040`
```